en/docs/Learn/APIGateway/MessageMediation/change-the-default-mediation-flow-of-api-requests.md

```diff
@@ -59,329 +59,192 @@ Let's see how to create a custom sequence using the WSO2 API Manager Tooling Pl
+
+!!! note
     Alternatively, you can create a CAR file including the sequences and can deploy it in API Manager. For more information, see [Deploying Composite Applications in the Server](https://docs.wso2.com/display/ADMIN44x/Deploying+Composite+Applications+in+the+Server) , which is in the WSO2 Administration Guide.
 
 
-17. Sign in to the API Publisher again, select the API that you created earlier.
-    ![](../../../assets/img/Learn/weatherapi-overview.png)
-18. Go to the API's **Runtime Configurations** tab, enable the **Message Mediation** switch and enable **Custom Policies**. Now you can upload the sequence that you created for the In flow. Next, click **Save** to engage the sequence to the API.
+15. Sign in to the API Publisher again, select the API that you created earlier.
 
-        !!! tip
-    **Tip** : It might take a few minutes for the sequence to be uploaded into the API Publisher. If it isn't there, please check again later.
+    ![](../../../assets/img/Learn/pizzashack-overview.png)
 
+16. Go to the API's **Runtime Configurations** tab, enable the **Message Mediation** switch and enable **Common Policies**. Now you can select the sequence that you created for the In flow. Next, click **Save** to engage the sequence to the API.
 
-    ![](../../../assets/img/Learn/upload-in-seq.png)
+    ![](../../../assets/img/Learn/pizzaorder-global-seq.png)
 
-        !!! note
+!!! note
     When selecting a mediator, make sure that it is a non-blocking mediator as blocking mediators are not supported in API Gateway custom mediations. For more details, see [Adding Mediation Extensions](/Users/isharacooray/Documents/apim/public/docs-apim/en/docs/Learn/Extensions/adding-mediation-extensions.md).
 
 
-19. Sign in to the API Store, subscribe to the API that you just published, and generate the access tokens in order to invoke the API.
-    ![](../../../assets/img/Learn/subscribe-weatherapi.png)
+17. Sign in to the API Developer Portal, subscribe to the API that you just published, by selecting the **Credentials** tab.
+
+    ![](../../../assets/img/Learn/pizzashack-subscribe.png)
+
+18. Select PROD KEYS,  generate the access tokens in order to invoke the API.
+
+    ![](../../../assets/img/Learn/pizzashack-prodkeys.png)
+
+19. Copy the generated access token.
+
+    !!! Tip
+        You may save it in a file for future use.
+
+    ![](../../../assets/img/Learn/pizzashack-copy-token.png)
 20. Click the **Try Out** tab of the API.
-    It opens the integrated API Console using which you can invoke the API. Click on the GET '/current/{country}/{zipcode}' resource and click on **Try it out**
-    ![](../../../assets/img/Learn/tryout-weatherapi.png)
-21. Give the following values for the parameters and invoke the API. You can also give any other value of your choice.
+    It opens the integrated API Console using which you can invoke the API. Paste the access token in the Access Token input box.
```

Comment:
merge the following with the previous sentence. "This opens the integrated API Console which can be used to invoke the API." Break the rest of it as a separate point.